zyte_spider_templates/spiders/article.py

```diff
@@ -286,15 +286,16 @@ def start_requests(self) -> Iterable[scrapy.Request]:
 
         for url in self.start_urls:
             meta = {"request_type": request_type}
-            yield self.get_parse_request(
-                ProbabilityRequest(
-                    url=url,
-                    name=f"[{request_type.value.name}]",
-                    metadata=ProbabilityMetadata(probability=probability),
-                ),
-                meta=meta,
-                is_feed=False,
-            )
+            with self._log_exception:
```

Comment:
Can we limit the exceptions which are logged, i.e. catch only the specific ones - something like this?

```
with self._log_exception(ValueError):
```

https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse indicates that urlparse fails with ValueError on incorrect URLs. I'm not confident it's the only exception which can be raised, who knows :)